pinotdb/db.py

```diff
@@ -179,7 +180,8 @@ def cursor(self):
         """Return a new Cursor Object using the connection."""
         if not self.session or self.session.is_closed:
             self.session = httpx.Client(
-                verify=self._kwargs.get('verify_ssl'))
+                verify=self._kwargs.get('verify_ssl'),
+                timeout=self._kwargs.get('timeout'))
```

Comment:
I think that we probably need to set

`timeout=self.timeout`? 

Does the following handle `None` properly? 
```
timeout=self._kwargs.get('timeout'))
```

Reply:
Yes it does, `self._kwargs.get('timeout'))` will return the timeout if provided else it will return `None`